src/App.spec.tsx

```diff
@@ -48,7 +56,8 @@ describe('App', () => {
 
 	const pages: Page[] = [
 		{ outlet: 'home', content: <Home /> },
-		{ outlet: 'blog', content: <Blog /> },
+		{ outlet: 'blog', content: <Blog />, args: [{ isExact: () => true }] },
+		{ outlet: 'blog', content: undefined, args: [{ isExact: () => false }] },
```

Comment:
Why do we need this inexact outlet for `blog` with no `content`? What does this do?

Reply:
By moving the individual blog page's route under `blog`, the parent gets triggered. So we need to check for the route to be exact.